dynawo/sources/Modeler/Common/DYNSubModel.cpp

```diff
@@ -484,6 +484,7 @@ SubModel::setParameterFromSet(ParameterModeler& parameter, const shared_ptr<para
                               const parameterOrigin_t& origin) {
   if (parametersSet) {
     const string& parName = parameter.getName();
+    // std::cout << parName << std::endl;
```

Comment:
not needed

Reply:
yes, sorry, wrong manipulation with git, removed